meson.build

```diff
@@ -10,24 +10,15 @@ cc = meson.get_compiler('c')
 enable_runtime = get_option('runtime')
 enable_builtin = get_option('builtin')
 
-# We need to know the CRT being used to determine what .lib files we need on
-# Visual Studio for dependencies that don't normally come with pkg-config files
-vs_crt = 'release'
-vs_crt_opt = get_option('b_vscrt')
-if vs_crt_opt in ['mdd', 'mtd']
-  vs_crt = 'debug'
-elif vs_crt_opt == 'from_buildtype'
-  if get_option('buildtype') == 'debug'
-    vs_crt = 'debug'
-  endif
-endif
+# We need to know the build type to determine what .lib files we need on Visual Studio
+# for dependencies that don't normally come with pkg-config files for Visual Studio builds
+buildtype = get_option('buildtype')
 
 notfound = dependency('', required : false)
 libidn2_dep = notfound
 libicu_dep = notfound
 libidn_dep = notfound
 libunistring = notfound
-libiconv = notfound
```

Comment:
Is this being removed again on purpose? This was just added in a previous PR #136 

Or was it removed by accident during conflict resolution?

Reply:
See comment below.